tasks/main.yml

```diff
@@ -32,6 +33,36 @@
     dest: "{{ item.value.dest }}"
   with_dict: "{{ openstax_hypothesis_role.services }}" 
 
+- name: check and see if the sql directory exists
+  stat:
+    path: "{{ openstax_hypothesis_role.application_dir }}/sql"
+  register: sql_home
+
+- name: Create the sql directory if it does not exist
+  file: 
+    path: "{{ openstax_hypothesis_role.application_dir }}/sql"
+    state: directory
+    mode: 0750
+    user: "{{ openstax_hypothesis_role.user }}"
+    group: "{{ openstax_hypothesis_role.group }}"
+  when: not sql_home.stat.exists
```

Comment:
Might be unnecessary to check that the dir exists... could just call the file module regardless and it won't do anything if it already exists and has the proper permissions etc